openid-4-verifiable-presentations-1_0.md

```diff
@@ -1494,7 +1494,7 @@ issuers in Self-Sovereign Identity ecosystems using TRAIN</title>
         </front>
 </reference>
 
-# Examples with Credentials in Various Formats {#alternative_credential_formats}
+# Usage with Various Credential Formats {#alternative_credential_formats}
 
 OpenID for Verifiable Presentations is Credential format agnostic, i.e., it is designed to allow applications to request and receive Verifiable Presentations and Verifiable Credentials in any format, not limited to the formats defined in [@!VC_DATA]. This section aims to illustrate this with examples utilizing different Credential formats. Customization of OpenID for Verifiable Presentation for Credential formats other than those defined in [@!VC_DATA] uses extension points of Presentation Exchange [@!DIF.PresentationExchange].
```

Comment:
I would replace this paragraph entirely with what we have in VCI - it's cleaner and also better alignment btw the specs.

```suggestion
This specification defines several extension points to accommodate the differences across Credential formats. Sets of Credential format specific parameters or claims referred to as Credential format identifiers are identified by the Credential format identifier and used at these extension points.

This section defines Credential Format Profiles for a few of the commonly used Credential formats. Other specifications or deployments can define their own Credential Format Profiles.
```

Reply:
I'm not entirely sure we have "several extension points"?

I don't think we have "Credential format specific parameters" either?

Reply:
sd-jwt_alg_values and kb-jwt_alg_values are probably examples of.
but my main point is that a word "illustrates" should be changed:

```suggestion
OpenID for Verifiable Presentations is Credential format agnostic, i.e., it is designed to allow applications to request and receive Verifiable Presentations and Verifiable Credentials in any Credential format. This section defines Credential Format Profiles for a few of the commonly used Credential formats. Other specifications or deployments can define their own Credential Format Profiles.```